internal/services/firewall/firewall_resource_test.go

```diff
@@ -194,6 +195,31 @@ func TestAccFirewall_withZones(t *testing.T) {
 	})
 }
 
+func TestAccFirewall_skuTier(t *testing.T) {
```

Comment:
we should be more explicit here
```suggestion
func TestAccFirewall_skuTierUpdate(t *testing.T) {
```